tests/sentry/tsdb/test_redis.py

```diff
@@ -25,8 +25,15 @@ def setUp(self):
             ),
             vnodes=64,
             enable_frequency_sketches=True,
+            hosts={
+                i - 6: {'db': i} for i in xrange(6, 9)
+            },
         )
 
+    def tearDown(self):
+        with self.db.cluster.fanout('all') as client:
```

Comment:
I guess [docs](http://rb.readthedocs.io/en/latest/#rb.Cluster.all) say it's the same exact thing, but can we do
```python
with self.db.cluster.all() as client:
```
here?